apex/transformer/testing/distributed_test_base.py

```diff
@@ -114,7 +114,7 @@ def _setup_pre_spawn(self) -> None:
 
         self._has_ucx_tls = "UCX_TLS" in os.environ
         if not self._has_ucx_tls:
-            os.environ["UCX_TLS"] = "tcp,cuda_copy"
+            os.environ["UCX_TLS"] = "tcp,cuda"
         print('os.environ[\"UCX_TLS\"] = {}'.format(os.environ["UCX_TLS"]))
```

Comment:
For correct usage with https://github.com/Sergei-Lebedev/torch_ucc/commits/topic/active_set
I suggest to add `UCC_TLS=ucp`
```suggestion
        print('os.environ[\"UCX_TLS\"] = {}'.format(os.environ["UCX_TLS"]))
        
        self._has_ucc_tls = "UCC_TLS" in os.environ
        if not self._has_ucc_tls:
            os.environ["UCC_TLS"] = "ucp"
        print('os.environ[\"UCC_TLS\"] = {}'.format(os.environ["UCC_TLS"]))
```

Reply:
nice catch